test/tcs_test/tcsmolten_salt_common_data.h

```diff
@@ -29,7 +29,7 @@ void tcsmolten_salt_default(ssc_data_t &data)
     //  The actual weather file used by SAM could change and thus change the UI output values (different input (i.e., weather file) -> different outputs)
 	int n1 = sprintf(solar_resource_path, "%s/test/input_cases/moltensalt_data/daggett_ca_34.865371_-116.783023_psmv3_60_tmy.csv", std::getenv("SSCDIR"));
 
-    ssc_data_set_string(data, "solar_resource_file", solar_resource_path);
+	ssc_data_set_string(data, "solar_resource_file", "C:/Users/kfung/Documents/SAM/sam_dev/SAM/deploy/solar_resource/daggett_ca_34.865371_-116.783023_psmv3_60_tmy.csv");
```

Comment:
Reference solar_resource_path like before, and update that variable if necessary, noting that the path should stay within the ssc repository.

Reply:
Matt, I believe I also made this path error within tcsmolten_salt_cases.h for <int tcsmolten_salt_Tucson_AZ(ssc_data_t &data)>.

Can you verify?

Reply:
What you did here is change where to find the weather file, from the path given by solar_resource_path to "C:/User/kfung/...", where solar_resource_path is "...test/input_cases/moltensalt_data..." . You made this change when it was found that the weather file had intrinsically changed and was causing some of the test values to change. Instead of changing which weather file to use, you instead just copied the weather file actually used in the UI (from the sam folder/repository) and overwrote the one pointed to here. You didn't need to make any changes to the code here, because you changed the actual weather file instead. So, you need to undo your code change in this instance, making the green highlighted line look like the red highlighted line.

Any other code changes you think you may have made will be highlighted in red or green like the above.

Reply:
Matt, this change from green to red triggers the original issue we had in the beginning of this project, of an incorrect output value being produced via the weather file. When the red line of code is implemented, the test fails. Back to our original example, when implemented annual_energy is evaluated at 5.77e8 instead of matching the SAM UI output value of 5.81e8.